analyzer/runtime/extract.go

```diff
@@ -53,6 +53,12 @@ type BlockTransactionData struct {
 	Body                    interface{}
 	To                      *apiTypes.Address // Extracted from the body for convenience. Semantics vary by tx type.
 	Amount                  *common.BigInt    // Extracted from the body for convenience. Semantics vary by tx type.
+	EVMEncryptedFormat      *int
+	EVMEncryptedPublicKey   *[]byte
+	EVMEncryptedDataNonce   *[]byte
```

Comment:
are we able to decode the format/nonces/publickey into a more concrete type here? Or can it vary?